src-backend/api/serializer.py

```diff
@@ -48,7 +50,9 @@ class Meta:
             'id',
             'display_index',
             'procedure',
-            'elements'
+            'elements',
+            'last_modified',
+            'created'
```

Comment:
we should have a common convention to have a comma on the last element of a list (in both JS and Python) so we don't have -1 +1 commits